test/suite/test_schema03.py

```diff
@@ -568,5 +568,4 @@ def check_entries(self, tc):
                 self.check_one(full_idxname, cursor, key, val)
             cursor.close()
             self.show_python("cursor.close()")
-if __name__ == '__main__':
-    wttest.run()
+
```

Comment:
Also needs checking. I think the pattern here is cases where the "if main" isn't separated from the earlier code by a line break.